### ISSUE TITLE
[Feature] use only application icon as window title

### ISSUE THREAD

**Issue by Zerogaku** [bakkeby/dwm-flexipatch#416]
When using the winicon patch is it possible to disable the title/text and keep
only the icon? I kinda imagined enabling BAR_WINICON_PATCH and disabling
BAR_WINTITLE_PATCH possibly doing that.

**Comment by bakkeby** [comment]
As-is no, but it wouldn't be hard to create a module that only shows the window icon.

The main issue with that is practicality; not all windows will have an icon and it would make it difficult to differentiate between, say, three terminal windows that has the same icon.

One compromise could be to use the fancybar behaviour (where it doesn't use more space than needed) and only show the icon for inactive windows and the icon + full title for the currently focused window.

**Comment by Zerogaku** [comment]
yeah it's more of an aesthetic thing than a functional thing, at least for me most of the apps i use have an icon, I'm guessing another solution is to manuallt add icons for apps that don't have them, or just use a default generic icon for applications that don't come with an icon.
